spec/rspec/mocks/test_double_spec.rb

```diff
@@ -48,6 +48,19 @@ module Mocks
           end
         end
       end
+
+      describe ".register" do
+        pending "able to register test doubles" do
+          user = double('User')
+          allow(user).to receive(:name).and_return("FooBar")
+
+          RSpec::Mocks::TestDouble.register user.class do |user|
```

Comment:
`RSpec::Mocks::TestDouble` does not have a `register` method.  The goal is register the it with `RSpec::Support::ObjectInspector`, right?